app/models/foreman_kubevirt/kubevirt.rb

```diff
@@ -272,6 +272,23 @@ def convert_memory_to_bytes(memory)
       convert_memory(memory, :b)
     end
 
+    def console(uuid)
+      vm = find_vm_by_uuid(uuid)
+      vnc_details = client.vminstances.get_vnc_console_details(vm.name, namespace)
+      token = Base64.encode64(vnc_details[:token]).delete!("\n").delete("==")
+      token_protocol =  "base64url.bearer.authorization.k8s.io.#{token}"
+      plain_kubevirt_protocol = "plain.kubevirt.io"
```

Comment:
perhaps define it as a constant + freeze and set it on line 286 without local variable ?